README.md

```diff
@@ -22,14 +22,14 @@ A function is provided so you can use your own bindist if you want.
 The best ones to use from gitlab are the `fedora27` artifacts.
 
 ```
-nur.repos.mpickering.ghc.mkGhc {  url = "https://gitlab-artifact-url.com"; hash = "sha256"; }
```

Comment:
I'm not very keen that `mkGhc` becomes more fiddly to use.

Reply:
I'm neither. Unless we want to overload `mkGhc`, we'll need to provide the `system` value somewhere to specify what we are actually building :-(

I'm happy for better syntax, but didn't manage to come up with any :-/